task.sql

```diff
@@ -1,17 +1,40 @@
 CREATE DATABASE ShopDB; 
 USE ShopDB; 
 
--- Create a table to store countries 
 CREATE TABLE Countries (
-    ID INT,
+    ID INT AUTO_INCREMENT,
     Name VARCHAR(50),
     PRIMARY KEY (ID)
 ) ENGINE=InnoDB;
 
--- Create a table for caching GeoIP data (Columns: ID, IP Range, CountryID)
+CREATE TABLE GeoIPCache (
+    ID INT AUTO_INCREMENT,
+    IPRange VARCHAR(50),
+    CountryID INT,
+    
+    PRIMARY KEY (ID),
+    FOREIGN KEY (CountryID) REFERENCES Countries (ID) ON DELETE CASCADE
+) ENGINE=Memory;
 
--- Create a table for storing product descriptions for different countries (Columns: ID, CountryID, ProductID, Description )
+CREATE TABLE ProductDescription (
+    ID INT AUTO_INCREMENT,
+    CountryID INT,
+    ProductID INT,
+    Description TEXT,
+    
+    PRIMARY KEY (ID)
+) ENGINE=MyISAM;
```

Comment:
The `ProductDescription` table uses the `MyISAM` storage engine, which does not support foreign key constraints. If you plan to add foreign keys in the future, consider using the `InnoDB` storage engine instead.